docs/extend/modular-resources/_index.md

```diff
@@ -110,9 +110,12 @@ For example:
 {{< tabs name="Base Model Modules" >}}
 {{% tab name="Go" %}}
 
-This example module code is adapted from the full demo module available on the [Viam GitHub](https://github.com/viamrobotics/rdk/blob/main/examples/customresources/models/mybase/mybase.go), and creates a singular modular resource implementing Viam's built-in Base API (rdk:service:base).
+This example custom model code (<file>mybase.go</file>) and module (<file>mymodule.go</file>) is adapted from the full demo module available on the [Viam GitHub](https://github.com/viamrobotics/rdk/blob/main/examples/customresources/models/mybase/mybase.go), and creates a singular modular resource implementing Viam's built-in Base API (rdk:service:base).
```

Comment:
```suggestion
The example custom model code (<file>mybase.go</file>) and module (<file>mymodule.go</file>) are adapted from the full demo module available on the [Viam GitHub repository](https://github.com/viamrobotics/rdk/blob/main/examples/customresources/models/mybase/mybase.go), and creates a singular modular resource implementing Viam's built-in Base API (rdk:service:base).
```

supernit: The above aligns plural (`the ... code ... and module are adapted`) and article (`the ... repo`). The article change also applicable to the Python tab, if you like!